# HG changeset patch
# User Rik <rik@octave.org>
# Date 1471221239 25200
#      Sun Aug 14 17:33:59 2016 -0700
# Node ID ca8b8f12bd169e9e2802691579ca91dfe7370ba4
# Parent  378f8c0de848c5acf66ffd74479218b71d3bad21
NEWS: Announce implementation of TitleFontSizeMultiplier, TitleFontWeight,
LabelFontSizeMultiplier properties.

* NEWS: Announce implementation of TitleFontSizeMultiplier, TitleFontWeight,
LabelFontSizeMultiplier properties.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -90,16 +90,23 @@ Summary of important user-visible change
 
  ** The textscan function is now built-in and is much faster and much
     more Matlab-compatible than the previous m-file version.
 
  ** Dialog boxes--errordlg, helpdlg, inputdlg, listdlg, msgbox,
     questdlg, and warndlg--now exclusively use Qt for rendering.
     Java based versions have been removed.
 
+ ** The axes properties "TitleFontSizeMultiplier" and "TitleFontWeight"
+    are now implemented which control the default appearance of text
+    created with title() appear.
+    The axes property "LabelFontSizeMultiplier" is now implemented
+    which controls the default appearance of text created with
+    xlabel(), ylabel(), or zlabel().
+
  ** The graphics property 'erasemode' has been hidden, and will
     eventually be removed.  This property has also been removed
     from Matlab, and was never implemented in Octave.
 
  ** The graphics property 'graphicssmothing' for figures now controls whether
     anti-aliasing will be use for lines.  The default is "on".
 
  ** The publish function allows easy publication of Octave script files
